card-unification.md

```diff
@@ -0,0 +1,90 @@
+# Card Unification
+
+  * [ ] Change card shape to object with `type`, `name`, `render` and optional `edit` properties
+  * [ ] Card's `type` is validated by renderer (dom renderer cannot render 'text', e.g.)
+  * [ ] Change arguments passed by editor to card's `render` (or `edit`) method
+    * [ ] single argument object with `name`, `buffer`, `isInEditor`, `options`, `editorEnv` and `payload` properties
```

Comment:
`isInEditor` -> `isEditing`? "in editor" seems funky.

I'm kind of inclined to put it on `env`, but I'll let is go :-p Just don't want to make this a junk drawer.